### DISCUSSION TITLE
location of log in production?

### DISCUSSION THREAD

**Discussion by ubrmensch** [megahertz/electron-log#324] [General]
Hi, 

I've tried googling this but for some reason I cannot find the answer.  The default location in non-production on my Mac is ~/Library/Logs/app-name.  But I cannot find where the location is in production.  If I package the app and run it on my Mac, do I need to set a specific file directory location, or is there a production location default?  It is definitely not writing to the default location in production.

**Comment by ubrmensch** [comment]
Apologies...looks like I am doing something dumb in my packaging. 

I used ipcRender to figure out app.getPath('logs') and it appears the app name changes to electron-react-boilerplate from the actual name of the app in package.json when the application is packaged.  I see the log file now and it is indeed in ~/Library/Logs, but just under a different the boilerplate app name.

**Reply by megahertz** [reply]
It depends on many aspects set by the boilerplate. If the default path selection doesn't work for you case you can set custom path through the resolvePath callback. The call `app.getPath('logs')` isn't used by the library, so another path may be used.
